api/views/user_views.py

```diff
@@ -212,6 +216,7 @@ class Logout(APIView):
     permission_classes = [IsAuthenticated]
 
     def post(self, request):
+        logger.info(f"User {request.user.id} logged out. Previous token: {request.auth}")
```

Comment:
Same there, even though we're deleting it.

Reply:
We should definitely store this elsewhere though, in case we have to track something else down further down the line. Will fix it for logging regardless.

Reply:
Could you please elaborate on what you mean here?

Reply:
I was thinking that it may be worthwhile to store the expired tokens that were usedby users in the past on some sort of rolling window (say 30 days) so that if we implement some sort of feature down the line that logs that information as part of logging the entirety of a request, we know who that token once belonged to. 

Although this might not be relevant and instead we can store older IDs with the same system to make it a safer form of logging.

Reply:
We can discuss this further in another meeting, it doesn't concern this issue.